### DISCUSSION TITLE
Linear progress starts over when value is updated

### DISCUSSION THREAD

**Discussion by FrenchGithubUser** [quasarframework/quasar#15586] [General - Components / Directives / etc]
When using the linear progress component, the value is set with the `value` attribute. This works fine (right amount, animation works etc.). But when the value is updated (for example from 0.2 to 0.3) the progress starts over as if it started from 0 again.

Is this normal behavior ? Is there a way to make it continue from where it was for the previous value ?

**Comment by pdanpdan** [comment]
make a codepen

**Comment by FrenchGithubUser** [comment]
I can't manage to reproduce it with a codepen, however, the project in which this problem happens is on github here : https://github.com/FrenchGithubUser/Hatt

Steps to reproduce : 

- Select one or more categories that have 10 or more websites (to have some time to see the problem)
- Search for something
- See the progress bar below the websites list

The component is located in `frontend/src/components/websites/SelectedWebsites.vue`

The value is displayed in the console, at the same time as returned (in the computed function) and only goes up, never down.

**Reply by pdanpdan** [reply]
It's hard to check just by inspecting the code, but I suppose you end up re-rendering the linear progress (maybe searching toggles between true and false) so it thinks it should animate as if first rendered.
Try to replace `v-if` with `v-show`

**Reply by FrenchGithubUser** [reply]
I removed the v-if completely but it still starts over. However, the progress looks like it sometimes only starts from a specific lower value, not necessarily 0

Btw, if you want to run the code, you can have a look at contributing.md, it just requires a few golang deps

**Reply by pdanpdan** [reply]
Transform what is in template for value in a computed and log inside the computed the values
See how they change

**Reply by FrenchGithubUser** [reply]
That's what I did, as I said, the value only increases

**Reply by pdanpdan** [reply]
What you can try is to set `key="something"` on linear progress, to force vue to reuse the component

**Reply by FrenchGithubUser** [reply]
I just tried, and the result stays the same
